.travis.yml

```diff
@@ -6,8 +6,21 @@ cache:
   directories:
    - $HOME/.m2
 
-# skip install
-install: true
+install:
```

Comment:
I think you're right about caching. The part that takes the longest is the app engine component install. ~30 seconds.